Triphouse-project/index.html

```diff
@@ -9,7 +9,40 @@
     <link rel="stylesheet" href="style.css"/>
 </head>
 <body>
```

Comment:
![2021-04-16_00h32_26](https://user-images.githubusercontent.com/19935417/114941975-87530480-9e4c-11eb-8d63-4ff664cdda6c.png)
Wrap into div and add style {max-width: 1440px, margin: 0px auto}

Reply:
Fixed.